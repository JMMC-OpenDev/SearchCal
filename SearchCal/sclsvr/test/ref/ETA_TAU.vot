version https://git-lfs.github.com/spec/v1
oid sha256:0e1802bb7a330a474708c78556e8ebe2b218b6874a950fb691baa8fb9bafeaa6
size 172178
