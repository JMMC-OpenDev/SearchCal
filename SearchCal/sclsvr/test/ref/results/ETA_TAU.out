version https://git-lfs.github.com/spec/v1
oid sha256:0af598a995402f07e2c69602108f29cd4f78d68a1d567fc151ab3a1c9b74740b
size 115970
